plugin/completion/lib_complete.py

```diff
@@ -223,7 +224,9 @@ def complete(self, completion_request):
         if complete_obj is None or len(complete_obj.results) == 0:
             completions = []
         else:
-            completions = Completer._parse_completions(complete_obj)
+            point = completion_request.get_trigger_position()
+            trigger = view.substr(point - 2) + view.substr(point - 1)
```

Comment:
You don't know at this point where the cursor has moved. Hardcoding those numbers is wrong.
CompletionRequest instance available here has a get_trigger_position() method.